tests/testGaussianProcess.py

```diff
@@ -45,21 +739,21 @@ def testTooManyNeighbors(self):
         dimen = 10                         # dimension of each point
         data = np.zeros((nData, dimen))
         fn = np.zeros(nData)
-        gg = gp.GaussianProcessD(data, fn, gp.SquaredExpCovariogramD())
+        gg = afwMath.GaussianProcessD(data, fn, afwMath.SquaredExpCovariogramD())
         test = np.zeros(dimen)
         sigma = np.empty(1)
         mu_arr = np.empty(1)
 
         with self.assertRaises(pex.Exception):
-             gg.interpolate(sigma, test, 2*nData)
+            gg.interpolate(sigma, test, 2*nData)
         with self.assertRaises(pex.Exception):
-             gg.interpolate(sigma, test, -5)
+            gg.interpolate(sigma, test, -5)
         with self.assertRaises(pex.Exception):
-             gg.selfInterpolate(sigma, 0, 2*nData)
+            gg.selfInterpolate(sigma, 0, 2*nData)
         with self.assertRaises(pex.Exception):
-             gg.selfInterpolate(sigma, 0, -5)
+            gg.selfInterpolate(sigma, 0, -5)
         with self.assertRaises(pex.Exception):
-             gg.selfInterpolate(sigma, -1, nData-1)
+            gg.selfInterpolate(sigma, -1, nData-1)
```

Comment:
Part of what made the reported bug difficult to track before was that while most of the time this line did raise an exception, it was actually not the intended exception.  Is there anyway within pex.Exception to check that the right exception is being raised?  If so, no need to implement that everywhere, but probably a good idea to implement it here where we no there was a problem in the past.